cmd/apiserver/app/server/options.go

```diff
@@ -41,6 +41,7 @@ type ServiceCatalogServerOptions struct {
 	EtcdOptions *EtcdOptions
 	// TPROptions are options for serving with TPR as the backing store
 	TPROptions *TPROptions
+	StopCh     <-chan struct{}
```

Comment:
I see that below. Why not create it inside `NewCommandServer` or even the apiserver's `main`?